content/attachments/c0ceaa82-e37c-4b86-a766-59751c6759db/c0ceaa82-e37c-4b86-a766-59751c6759db-příloha.ttl

```diff
@@ -1,13 +1,12 @@
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
-<https://slovník.gov.cz/datový/pracovní-prostor/pojem/přílohový-kontext/instance342937355/diagram/4d828587-f0ca-44e0-8735-4797ee304146>
```

Comment:
"Přílohový kontext" should not be exported here at all. What we need is only it's content.

The attachment ID should be generic, and the variable part should be the same as the file name so that it is possible to find the attachment in a file.

Reply:
I am OK with either. Even <http://onto.fel.cvut.cz/ontologies/slovník/agendový/popis-dat/pojem/příloha/4d828587-f0ca-44e0-8735-4797ee304146> would be OK as well IMO.

Reply:
Agreed. The point is that the IRI should be attachment type agnostic.

Reply:
Solved:
https://github.com/opendata-mvcr/ssp/blob/master/content/attachments/instance-bff068bf-d294-4e8d-bd66-468169d10db2/instance-bff068bf-d294-4e8d-bd66-468169d10db2-p%C5%99%C3%ADloha.ttl